src/parser/UnicodeIdentifierTables.h

```diff
@@ -0,0 +1,55 @@
+/* * Copyright (c) 2020-present Samsung Electronics Co., Ltd
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ *  USA
+ */
+
+
+/* This file is automatically generated by the gen_unicode.py script
+* from https://www.unicode.org/Public/13.0.0/ucd/DerivedCoreProperties.txt
+* DO NOT EDIT!
+*/
+
+#ifndef __EscargotUnicodeIdentifierTables__
+#define __EscargotUnicodeIdentifierTables__
+
+namespace Escargot {
+
+/* Starting codepoints of identifier ranges. */
+static const uint16_t identRangeStart[532] = {
```

Comment:
Declaring static arrays is a header is not the best practice. You should create a cpp file with these array definitions, and this header should contain only the `extern` definition of these arrays.